docker/local-llm/Makefile

```diff
@@ -0,0 +1,5 @@
+.PHONY: build_docker
+
+docker:
+	docker build -t ragstack-basic .
```

Comment:
local-llm ?

Reply:
This is what Ben suggested to support use case that huggingface to download a llm to docker image. Does that make sense?

Reply:
yeah I meant to use it in the docker command 😃 
I updated the code for you